pkgs/applications/video/gnome-twitch/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, meson, cmake, ninja, wrapGAppsHook
+, libpeas, gtk3, libsoup, json-glib, webkitgtk
+, python
+, substituteAll
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-twitch";
+  version = "unstable-2019-05-20";
+
+  src = fetchFromGitHub {
+    owner = "vinszent";
+    repo = "gnome-twitch";
+    rev = "8e774a04fdd15fa4c9990918bf89295a232d1715";
+    sha256 = "0pvxfwbv4fms2d0sp43jw83kp1i6gq03kg6gwy8b1fhp473gxrfj";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson cmake ninja python wrapGAppsHook ];
+  buildInputs = [ libpeas gtk3 libsoup json-glib webkitgtk python ];
```

Comment:
Where is `python` needed in the program? Also please add `glib` as it for needs this library, even if it's propagated.